charts/datadog/values.yaml

```diff
@@ -96,6 +96,21 @@ datadog:
     ## ref: https://docs.datadoghq.com/agent/cluster_agent/clusterchecksrunner?tab=helm
     useClusterCheckRunners: false
 
+    # datadog.kubeStateMetricsCore.labelsAsTags -- Extra labels to collect from resources and to turn into datadog tag.
+    ## It has the following structure:
+    ## labelsAsTags:
+    ##   <resource1>:        # can be pod, deployment, node, etc.
+    ##     <label1>: <tag1>  # where <label1> is the kubernetes label and <tag1> is the datadog tag
```

Comment:
I think it's worth adding a concrete example, because the the label must match the transformation done by ksm, example:  `tags.datadoghq.com/version` becomes `label_tags_datadoghq_com_version`